keycloakClient/keycloakClient.go

```diff
@@ -135,11 +144,16 @@ func (t *tokenJSON) toTokenInfo(iat time.Time) *TokenInfo {
 func (c *Client) GetTokenInfo(realm string, username string, password string, force bool) (*TokenInfo, error) {
 	var newTokenInfo *TokenInfo
 	var err error
-	key := realm + username
```

Comment:
here is the original key that has been replaced with a struct that will prevent exceptionally clever or (un)lucky users from colliding and possibly getting a token they shouldn't have.